src/bridge/LibcoreBridge.js

```diff
@@ -120,41 +120,48 @@ const LibcoreBridge: WalletBridge<Transaction> = {
   },
 
   synchronize: account =>
-    libcoreSyncAccount.send({ rawAccount: encodeAccount(account) }).pipe(
-      map(rawSyncedAccount => {
-        const syncedAccount = decodeAccount(rawSyncedAccount)
-        return account => {
-          const accountOps = account.operations
-          const syncedOps = syncedAccount.operations
-          const patch: $Shape<Account> = {
-            id: syncedAccount.id,
-            freshAddress: syncedAccount.freshAddress,
-            freshAddressPath: syncedAccount.freshAddressPath,
-            balance: syncedAccount.balance,
-            blockHeight: syncedAccount.blockHeight,
-            lastSyncDate: new Date(),
-          }
-
-          const hasChanged =
-            accountOps.length !== syncedOps.length || // size change, we do a full refresh for now...
-            (accountOps.length > 0 &&
-              syncedOps.length > 0 &&
-              (accountOps[0].accountId !== syncedOps[0].accountId ||
-              accountOps[0].id !== syncedOps[0].id || // if same size, only check if the last item has changed.
-                accountOps[0].blockHeight !== syncedOps[0].blockHeight))
-
-          if (hasChanged) {
-            patch.operations = syncedAccount.operations
-            patch.pendingOperations = [] // For now, we assume a change will clean the pendings.
-          }
-
-          return {
-            ...account,
-            ...patch,
+    libcoreSyncAccount
+      .send({
+        accountId: account.id,
+        freshAddressPath: account.freshAddressPath,
```

Comment:
why do we need `freshAddressPath` to sync?

Reply:
to extract out some info: segwit? split? etc..